xrviz/dashboard.py

```diff
@@ -1,7 +1,6 @@
 import panel as pn
 import xarray as xr
 import hvplot.xarray
-from cartopy import crs as ccrs
```

Comment:
`cartopy` is not being used anywhere in the master. 
`pip install cartopy` does not works. The only way to install it is using conda or building from source as stated in [Installing Cartopy — cartopy 0.17.0 documentation](https://scitools.org.uk/cartopy/docs/latest/installing.html) . In order to use projection panel, installation using conda would be the only way.

Reply:
OK, so the projection panel should be unavailable if cartopy is not installed, and we need to decide whether to include it in the conda requirements or not.